Layout/Shared/ThemeButton/index.js

```diff
@@ -3,7 +3,7 @@ import { Switch } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 
 import { useDarkMode } from 'contexts/ui-context';
-import { AnalyticCategories, createEvent, analytics } from "utils/analytics";
+import { AnalyticCategories, createEvent, useAnalytics } from "contexts/analytics";
```

Comment:
Working on following the impace of the change from analytics to UseAnalytics with a new folder as reference.